### PR TITLE
Provide a means to edit the layout styles of tabs more explicitly & easily

### PR DESCRIPTION
**Pull request by lozandier**
## PR Goals
- Enable more easier handling of the alignment & styling of the direct container of tabs
- Be more consistent with the style hook provided by `paper-tab` to its container of the content slotted in (`--paper-tab-content`)

## Side-effects
- New `--paper-tabs-content` custom property set and its corresponding `@apply` hook
- The file is now 5 lines bigger.

## Estimated Review Time
- 5-10 minutes

## Recommended reviewers
@bicknellr?

### CODE REVIEW

File: paper-tabs.html [hunk 1]
```diff
@@ -125,14 +127,17 @@
         height: 100%;
         white-space: nowrap;
         overflow: hidden;
```
Comment (lozandier) [inline paper-tabs.html]: An alternative would be for this to *not* be hardcoded, but use `inherit`

File: paper-tabs.html [hunk 2]
```diff
@@ -125,14 +127,17 @@
...
+        display: flex;
```
Comment (bicknellr) [inline paper-tabs.html]: (Reminder about this stray `display: flex` that you pointed out earlier.)

Comment (lozandier) [inline paper-tabs.html]: Done.

**Comment by bicknellr** [comment]
(from external discussion) So instead of merging this PR, given that the intended use case here is more related to paper-tabs' appearance than its behavior, I'm working on pulling out the standalone selection bar thing from the shop demo (['shop-tabs-overlay'](https://github.com/Polymer/shop/blob/master/src/shop-tabs-overlay.html)) into its own repo. I'll leave this PR open to act as the bug / if for some reason the element doesn't work out, then this will still be an applicable workaround.

**Comment by jimivdw** [comment]
@bicknellr any news on this? For my project, I have to override some styling in the `#tabsContainer`, so the `--paper-tabs-container` mixin would make my life a lot easier :-).

**Comment by bicknellr** [comment]
Getting the shop-tabs-overlay equivalent out the door hasn't happened yet and is stalling resolving this PR for no real reason. The tests have failed here because of a lint error in paper-icon (which is  incorrect) but I've manually run this PR through WCT in saucelabs and everything checks out, so LGTM.

One problem this PR is meant to solve falls under the same stacking-context issue that a lot of other stuff runs into. ('dialogs', 'pop-up menus') There's not much that can be done about this right now since it relies on lower-level changes to the rendering engine / how CSS works. With this PR you'll be able to change `overflow` for the two elements in paper-tabs that would prevent elements from being visible outside the bounds of the tabs but scrolling tabs will break as a result. With no platform-level fix in the near term, it's probably best to leave this up to the user with a 'use at your own risk' policy.

**Pull request merged**